apps/web/src/components/(dashboard)/avatar/stack-avatars-ui.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { useWindowSize } from '@documenso/lib/client-only/hooks/use-window-size';
+import type { Recipient } from '@documenso/prisma/client';
+import { Popover, PopoverContent, PopoverTrigger } from '@documenso/ui/primitives/popover';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@documenso/ui/primitives/tooltip';
+
+import { StackAvatars } from './stack-avatars';
+import { StackAvatarsComponent } from './stack-avatars-component';
+
+export type StackAvatarsUIProps = {
+  recipients: Recipient[];
+  position?: 'top' | 'bottom';
+  children?: React.ReactNode;
+};
+
+export const StackAvatarsUI = ({ recipients, position, children }: StackAvatarsUIProps) => {
+  const size = useWindowSize();
+
+  return (
```

Comment:
I believe you could avoid the ternary operator here, and/or perhaps create smaller components:
```
if (size.width ? 1050) {
  return <ToolTipStack recipients={recipients} />
}
return <PopoverStack recipients={recipients} />
```